docs/user/production-considerations/production.asciidoc

```diff
@@ -51,11 +51,17 @@ Settings that must be the same:
 [source,js]
 --------
 xpack.security.encryptionKey //decrypting session information
+xpack.security.authc.providers // authentication providers
 xpack.reporting.encryptionKey //decrypting reports
 xpack.encryptedSavedObjects.encryptionKey // decrypting saved objects
 xpack.encryptedSavedObjects.keyRotation.decryptionOnlyKeys // saved objects encryption key rotation, if any
 --------
 
+[WARNING]
+====
+If authentication providers do not match, sessions from unrecognized providers in each {kib} instance will be deleted during that instance's cleanup. This also applies to any {kib} instances that are backed by the same {es} instance and share the same kibana.index, even if they are not behind the same load balancer.
```

Comment:
__nit:__ if we include `xpack.security.session.*` above, we might need to adjust this warning to mention out-of-sync session config as well.

__optional nit:__ it might be just me, so feel free to ignore, but `instance's cleanup` sounds a bit confusing to me, maybe `instance's regular session cleanup` or something along these lines to clarify what type of cleanup we mean?